clients/admin-ui/src/types/api/models/BasicSystemResponse.ts

```diff
@@ -79,6 +79,10 @@ export type BasicSystemResponse = {
    * If specified, the unique identifier for the vendor that was previously associated with this system.
    */
   previous_vendor_id?: string;
+  /**
+   * The deleted date of the vendor that's associated with this system.
+   */
+  vendor_deleted_date?: string;
```

Comment:
side q: the `vendor_deleted_date` will always be in the response, right? It'll just be `null` by default?

(this aspect of the autogenerated types bugs me)

Reply:
ah yeah that's correct. idk the best way to handle that in typescript, let me know if this is something you think is necessary to adjust